frontend/packages/console-shared/src/utils/utils.ts

```diff
@@ -26,3 +26,10 @@ export const createLookup = <A extends K8sResourceKind>(
   }
   return {};
 };
+
+export const getRandomChars = (digit = 6): string => {
+  return Math.random()
+    .toString(36)
+    .replace(/[^a-z0-9]+/g, '')
+    .substr(1, digit);
+};
```

Comment:
nit: Not new, but digits refer to numbers. I'd suggest `len`

```suggestion
export const getRandomChars = (len = 6): string => {
  return Math.random()
    .toString(36)
    .replace(/[^a-z0-9]+/g, '')
    .substr(1, len);
};
```